packages/@react-types/actiongroup/src/index.d.ts

```diff
@@ -13,16 +13,30 @@
 import {DOMProps, ItemElement, ItemRenderer, MultipleSelection, StyleProps} from '@react-types/shared';
 
 export interface ActionGroupProps<T> extends DOMProps, StyleProps, MultipleSelection {
+  /**
+   * The axis the ActionGroup should align with.
+   * @default 'horizontal'
+   */
   orientation?: 'horizontal' | 'vertical',
   children: ItemElement<T> | ItemElement<T>[] | ItemRenderer<T>,
+  /**
+   * Whether the ActionGroup is disabled or not.
+   * Shows that a selection exists, but is not available in that circumstance.
+   */
   isDisabled?: boolean
 }
 
 export interface SpectrumActionGroupProps<T> extends ActionGroupProps<T> {
+  /** Whether the ActionButtons should be displayed with a [emphasized style](https://spectrum.adobe.com/page/action-button/#Emphasis). */
```

Comment:
i wasn't sure how to phrase this -- other options include saying something like `Whether the items should...`.

someone should let me know if they have an opinion.